api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimService.java

```diff
@@ -18,31 +22,42 @@
     private final EligibilityClient client;
     private final EligibilityStatusCalculator eligibilityStatusCalculator;
 
+    private final Map<EligibilityStatus, ClaimStatus> eligibilityStatusMapper = ImmutableMap.of(
```

Comment:
This might be better called 'statusMap'?

Reply:
Renamed.